cfgov/jinja2/v1/youth_employment_success/index.html

```diff
@@ -7,7 +7,7 @@
 
 {% block content %}
 
-<main class="content content__1-3" role="main">
+<main class="no-js content content__1-3" role="main">
```

Comment:
Is the `no-js`/`js` class on the `html` element not working how you want it?

Reply:
I don't know how to add it to the `html` element when I'm extending an existing layout 😬 . All the content for the tool exists under the `main` element, so it seemed like an ok solution!

Reply:
You have the functionality of having it added to the html element already. (`js` class appears on `html`, or `no-js` in things like IE10). However, I'm wondering if it's not working how you need it to be working?

Reply:
Oh I see you've answered it below.